datafusion/execution/src/object_store.rs

```diff
@@ -60,6 +60,11 @@ impl ObjectStoreUrl {
     pub fn as_str(&self) -> &str {
         self.as_ref()
     }
+
+    /// Returns as Url
+    pub fn as_url(&self) -> &Url {
```

Comment:
This is not required, since there is another accessor available.

Reply:
What is the accessor? `AsRef<str>`?

Reply:
`AsRef<Url>`